sysbuild/Kconfig.suit

```diff
@@ -164,6 +206,66 @@ config SUIT_ENVELOPE_APP_RECOVERY_ARTIFACT_NAME
 	help
 	  Name of the root SUIT artifact.
 
+config SUIT_ENVELOPE_APP_RECOVERY_SIGN
+	bool "Sign the target envelope"
```

Comment:
nit: `target` -> `recovery`?

Reply:
Changed it locally; If no groundbreaking review changes are requested I will push together with the manifest update